docs/user/dashboard/lens.asciidoc

```diff
@@ -48,389 +13,144 @@ To view the data summary for a field, click *i* next to the field.
 
-To compare the average of bytes transfer to the number of users that visit your website, add a line chart layer.
+The percentages for each value are computed for each bucket with the division between the metric for 
+the value, and the metric of all documents. For example, the percentage for date histogram is calculated 
+for each time bucket. If the bucket has 0 documents, then the percentage chart displays 0%, not a 100% value.
 
-. From the editor, click *+*.
-+
-[role="screenshot"]
-image::images/lens_end_to_end_3_2.png[Add new layer button]
+When you use a bucketed metric, such as *Top values*, you can display the <<what-is-the-other-category, *Other* field>> 
+in the percentage chart. When you display the *Other* field, the final result displays the correct ratio.
 
-. From the new layer editor, click the *Chart type* dropdown, then click the line chart.
-+
 [role="screenshot"]
-image::images/lens_end_to_end_3_3.png[Change layer type]
-+
-The chart type for the visualization changes to *Mixed XY*.
-
-. From the *Available fields* list, drag and drop the data fields to the *Drop a field or click to add* fields in the editor.
-
-.. Drag and drop *timestamp* to the *Horizontal axis* field.
-
-.. Drag and drop *clientip* to the *Vertical axis* field.
-
-. Change the *timestamp* interval.
-
-.. From the editor, click *timestamp* in the line chart layer.
-
-.. Select *Customize time interval*.
-
-.. Change the *Minimum interval* to `1 days`, then click *Close*.
-
-. Change the *Unique count of clientip* label and color.
-
-.. From the editor, click *Unique count of clientip*.
-
-.. In the *Display name* field, enter `Unique visitors` in the line chart layer.
-
-.. In the *Series color* field, enter *#CA8EAE*, then click *Close*.
-
-[discrete]
-[[configure-the-multiaxis-chart]]
-==== Configure the y-axes
-
-There is a significant difference between the *timestamp per day* and *Unique visitors* data, which makes the *Unique visitors* data difficult to read. To improve the readability, 
-display the *Unique visitors* data along a second y-axis, then change the formatting. When functions contain multiple formats, separate axes are created by default. 
-
-. From the editor, click *Unique visitors* in the line chart layer.
-
-. For *Axis side*, click *Right*, then click *Close*.
+image::images/lens_area_percentage.png[Lens Area percentage]
 
 [float]
-[[change-the-visualization-type]]
-==== Change the visualization type
+[[can-i-show-value-labels-for-my-chart]]
+===== How do I display value labels?
 
-. From the editor, click *Average of bytes* in the area chart layer. 
+A subset of *Lens* visualizations support value labels. Value labels are unsupported in date histogram.
 
-. From the *Value format* dropdown, select *Bytes (1024)*, then click *Close*.
+* *Bar* and *Horizontal Bar*
 +
 [role="screenshot"]
-image::images/lens_end_to_end_3_4.png[Multiaxis chart]
+image::images/lens_value_labels_xychart_toggle.png[Lens Bar chart value labels menu]
```

Comment:
There's no text-based explanation of how to open this menu, can you add one above?